lib/cisco_node_utils/cmd_ref/vrf.yaml

```diff
@@ -64,8 +64,27 @@ route_distinguisher:
   _exclude: [ios_xr]
   get_value: '/^rd (\S+)$/'
   set_value: '<state> rd <rd>'
+  context: ['vrf context <vrf>']
   default_value: ''
 
+route_policy_export:
+  _exclude: [nexus]
```

Comment:
FWIW it looks like the Nexus equivalent CLI is:

```
vrf context <vrf>
 address-family <afi> <safi>
  export map <policy_name>
```

Similarly for 'route-policy import' --> 'import map'